packages/account/src/Components/poa/status/submitted/submitted.tsx

```diff
@@ -1,14 +1,20 @@
-import { Icon, Text } from '@deriv/components';
-import { localize } from '@deriv/translations';
 import React from 'react';
 import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
 import { PlatformContext } from '@deriv/shared';
-import { ContinueTradingButton } from 'Components/poa/continue-trading-button/continue-trading-button.jsx';
+import { TPlatformContext } from 'Types';
+import { ContinueTradingButton } from 'Components/poa/continue-trading-button/continue-trading-button';
 import PoiButton from 'Components/poi/poi-button';
 import IconMessageContent from 'Components/icon-message-content';
 
-export const Submitted = ({ needs_poi, is_description_enabled = true }) => {
-    const { is_appstore } = React.useContext(PlatformContext);
+type TSubmitted = {
+    needs_poi: boolean;
```

Comment:
we have the same type in the `poa/status/needs-review/needs-review.tsx` file you can export it from there